index.html

```diff
@@ -14,12 +14,12 @@
         <a href="index.html"><img src="assets/images/charcoal.png" height="64" title="Charcoal: spam-crusaders of the Internet" /></a>
       </div>
       <div class="navigation text-center">
-        <div><a href="#">home</a></div>
-        <div><a href="https://metasmoke.erwaysoftware.com/">metasmoke</a></div>
```

Comment:
I did this for a reason: the linebreak + indentation between nav items causes a whitespace to render between nav items on the page, offsetting them from the center slightly. This fixes that.